plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarResource.java

```diff
@@ -73,6 +74,23 @@
 	 */
 	private String errorLogger = "An error occurred.";
```

Comment:
Try avoiding general error messages and be as specific as possible

Reply:
fixed